app/views/eligibility_interface/start/show.erb

```diff
@@ -1,44 +1,61 @@
+  <li class="govuk-body">an undergraduate degree equal to a UK bachelor’s degree</li>
+  <li class="govuk-body">completed teacher training equal to a <%= govuk_link_to "level 6 qualification", "https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels#:~:text=Level%206%20qualifications%20are%3A,graduate%20certificate" %></li>
+  <li class="govuk-body">completed all mandatory induction processes to gain full teacher registration in the country where you qualified</li>
+  <li class="govuk-body">been recognised as a professional teacher in the country where you qualified, and have no conditions or restrictions on your teaching practice</li>
+  <li class="govuk-body">qualifications to show you can teach children aged anywhere between 5 and 16 years old</li>
+  <li class="govuk-body">at least one school year (9 months) of experience employed as a teacher in any country, but must be from after you first became qualified to teach</li>
 </ul>
 
-<p class="govuk-body">We’ll ask some questions to check your eligibility to apply for QTS in England. This should take about 5 minutes.</p>
+<p class="govuk-body">Depending on the country that you trained to teach in, you may be required to have:</p>
 
-<p class="govuk-body">There’s no charge to apply for QTS. You do not need to be in the UK to use this service.</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li class="govuk-body">qualifications to show you can teach children aged anywhere between 11 and 16 years old</li>
+  <li class="govuk-body">a teaching specialism in one of the following subjects: maths, science, biology, chemistry, physics, French, German, Italian, Japanese, Latin, Mandarin, Russian, or Spanish</li>
+</ul>
 
-<% if FeatureFlags::FeatureFlag.active?(:teacher_applications) %>
-  <p class="govuk-body">
-    If you’ve already started your application using this service, you can
-    <%= govuk_link_to "sign in to continue working on it", :create_or_new_teacher_session %>.
-  </p>
-<% end %>
+<p class="govuk-body">If you trained in England or Wales, or have not yet qualified, learn more about <%= govuk_link_to "other routes to getting QTS", "https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#routes-to-qts" %>.</p>
 
 <%= form_with url: eligibility_interface_start_path, method: :post do |f| %>
   <button type="submit" formnovalidate="formnovalidate" class="govuk-button govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-9" data-module="govuk-button" data-prevent-double-click="true">
-    Start now
+    Check you’re eligible
     <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" focusable="false" aria-hidden="true"><path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"></path></svg>
   </button>
```

Comment:
We can change this to the `govuk_start_button` component in a form that didn't exist originally: https://govuk-components.netlify.app/components/start-button/#rendering-a-start-button-in-a-form

```rb
govuk_start_button(text: "Check you’re eligible", as_button: true)
```